testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -99,7 +99,10 @@ class ProfileTestHelper @Inject constructor(
   /** Login to user profile. */
   fun loginToUser() = logIntoProfile(internalProfileId = 1)
 
-  /** Login to a new user profile, since profile #2 is not used as a default user anywhere in this helper. */
+  /**
+   * Login to a new user profile that has no progress for any topics or stories. This relies on other
```

Comment:
Super nit: 'Log into' since that's the verb form.

Reply:
Done